omc3/run_kmod.py

```diff
@@ -33,6 +33,9 @@ def kmod_params():
                          type=str,
                          required=True,
                          help='path to working directory with stored KMOD measurement files',)
+    parser.add_parameter(name='outputdir',
+                         type=str,
```

Comment:
why not `type=pathlib.Path` ?

Reply:
switching everything to Pathlib (and testing everything in the background behaves the same) is a separate issue #238 and i think not in the spirit of what this quick fix PR is supposed to do